src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -157,22 +164,27 @@ define(function (require) {
     };
 
     SegmentedReq.prototype._createQueue = function () {
+      var self = this;
```

Comment:
I agree that we should flag the queue as created inside createQueue itself.  Good catch.

I still think it makes sense to default _queueCreated to false during instantiation though since it can technically be leveraged (via reportStatus) prior to start() being invoked.